lib/babe/crypto.go

```diff
@@ -84,6 +84,37 @@ func checkPrimaryThreshold(randomness Randomness,
 	return inoutUint.Compare(threshold) < 0, nil
 }
 
+func claimSecondarySlotVRF(randomness Randomness,
```

Comment:
this is interesting, the VRF output isn't compared to any sort of threshold? I'm wondering what the point of the VRF is.. could you link to the substrate code? just wondering if they have any explanation of this

Reply:
@kishansagathiya not something that needs to be changed code-wise, but just curious!

Reply:
It seems like this vrf can be consumed by parachains, I am not sure why. https://github.com/paritytech/substrate/blob/master/client/consensus/babe/README.md

Reply:
This is the relevant code 
https://github.com/paritytech/substrate/blob/c22fce5a311beede13479c9a00cca85d823b6b00/client/consensus/babe/src/authorship.rs#L158-L166